app/controllers/application_controller.rb

```diff
@@ -2,10 +2,21 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :configure_permitted_parameters, if: :devise_controller?
 
+  #find last universal ancestor
+  helper_method :find_lua
+
   protected
 
-def configure_permitted_parameters
-  devise_parameter_sanitizer.permit(:sign_up, keys: [:name, :surname, :nickname])
-end
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name, :surname, :nickname])
+  end
+
+  def find_lua(object)
```

Comment:
Zdaje się, że (przynajmniej na razie) korzystasz z tego tylko w `CommentsController` - skoro tak, to raczej tam powinna być ta metoda